types/connect-sqlite3/index.d.ts

```diff
@@ -2,169 +2,180 @@
 // Project: https://github.com/rawberg/connect-sqlite3
 // Definitions by: oof2win2 <https://github.com/oof2win2>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.2
 
 import * as express from "express";
 import { EventEmitter } from "events";
 
-interface SQLiteStoreOptions {
-	/**
-	 * Table of where sessions are stored
-	 * @default "sessions"
-	 */
-	table?: string;
-
-	/**
-	 * Database file name
-	 * @default "sessionsDB"
-	 */
-	db?: string;
-
-	/**
-	 * Directory where to save database in
-	 * @default "."
-	 */
-	dir?: string;
-
-	/**
-	 * Enables [WAL](https://www.sqlite.org/wal.html) mode (defaults to false)
-	 * @default "false"
-	 */
-	concurrentDB?: string;
+declare namespace connect {
+	interface SQLiteStoreOptions {
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/connect-sqlite3/index.d.ts;
  Line: 11;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.